# patch from debian svn
## 50_add_syspath.dpatch by Thomas Weber <thomas.weber.mail@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Put library path in loadpath (bug fix for #477556)
## DP: http://www.cae.wisc.edu/pipermail/octave-maintainers/2008-May/007330.html
diff -urNad octave3.0-3.0.1~/src/load-path.cc octave3.0-3.0.1/src/load-path.cc
--- octave3.0-3.0.1~/src/load-path.cc	2008-04-21 18:00:20.000000000 +0200
+++ octave3.0-3.0.1/src/load-path.cc	2008-05-21 10:20:38.000000000 +0200
@@ -382,7 +382,7 @@
     xpath += dir_path::path_sep_str + tpath;
 
   if (! sys_path.empty ())
-    xpath += sys_path;
+    xpath += dir_path::path_sep_str + sys_path;
 
   do_set (xpath, false);
 }
diff -urNad octave3.0-3.0.1~/src/load-path.cc.orig octave3.0-3.0.1/src/load-path.cc.orig
